app/src/main/java/org/oppia/app/profile/AddProfileFragment.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.app.profile
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that allows users to create new profiles */
+class AddProfileFragment : InjectableFragment() {
+  @Inject lateinit var addProfileFragmentPresenter: AddProfileFragmentPresenter
+
+  override fun onAttach(context: Context?) {
+    super.onAttach(context)
+    fragmentComponent.inject(this)
+  }
+
+  override fun onCreateView(
+    inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
```

Comment:
This can be converted to a single line:

`override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {`